.github/workflows/test.yml

```diff
@@ -18,9 +21,9 @@ jobs:
       postgres:
         image: postgres:12
```

Comment:
Hey @foarsitter :),
What you think about using the alpine image instead ? It will speed things up a bit

Reply:
Never thought about it but will definitely be an improvement.